index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <meta name="author" content="Mavericks Balitaan" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="headerText">Welcome</div>
+          <div class="menu">
+            <img src="./svg/hamburger.svg" alt="menu" />
+          </div>
+        </div>
+      </header>
+      <div class="headlineSection">
+        <div class="headlineTextGroup">
+          <div class="headlineText">Hey there. I'm Mavericks</div>
+          <div class="headlineText2">I'm a software developer</div>
```

Comment:
- [ ] Kindly use `h1` instead of `div` here `<div class="headlineText">Hey there. I'm Mavericks</div>` because this is your main header.
- [ ] Kindly use `hx` like `h2/h3` instead of `div` here `<div class="headlineText2">I'm a software developer</div>` because this is a secondary header.

It's more semantic to use the header tags with headers on your page.